Python/Product/Cookiecutter/CookiecutterToolWindow.cs

```diff
@@ -209,21 +209,19 @@ sealed class CookiecutterToolWindow : ToolWindowPane, IVsInfoBarUIEvents {
             _uiShell.UpdateCommandUI(0);
         }
 
-        private void OpenGeneratedFolder(string folderPath) {
-#if DEV15_OR_LATER
-            OpenInSolutionExplorer(folderPath);
-#else
-            OpenInWindowsExplorer(folderPath);
+        private void ExecuteCommand(string name, string args) {
+            try {
+                _dte.ExecuteCommand(name, args ?? string.Empty);
```

Comment:
We've definitely hit issues here in the past. I'd rather not risk it. (Also, localization may not be 100% complete for VS 2017 yet...)